girder/api/sftp.py

```diff
@@ -261,32 +262,29 @@ def shutdown_request(self, request):
         pass
 
 
-def _main():  # pragma: no cover
+@click.command(name='sftpd', short_help='Run the Girder SFTP service.',
+               help='Run the Girder SFTP service.')
+@click.option('-i', '--identity-file', show_default=True,
+              default=os.path.expanduser(os.path.join('~', '.ssh', 'id_rsa')),
+              help='The identity (private key) file to use')
+@click.option('-H', '--host', show_default=True, default='localhost',
+              help='The interface to bind to')
+@click.option('-p', '--port', show_default=True, default=DEFAULT_PORT, type=int,
+              help='The port to bind to')
+def _main(identity_file, port, host):  # pragma: no cover
```

Comment:
Could you make this `main` then, for consistency? I agree we can still consider it excluded from the API surface.

Reply:
Done.